src/actions/breakpoints.js

```diff
@@ -95,15 +95,25 @@ export function syncBreakpoint(
  * @param {Boolean} $1.disabled Disable value for breakpoint value
  */
 
-export function addBreakpoint(location: Location, condition: ?string) {
-  const breakpoint = createBreakpoint(location, { condition });
+export function addBreakpoint(
+  location: Location,
+  condition: ?string,
+  hidden: ?boolean
+) {
+  const breakpoint = createBreakpoint(location, { condition, hidden });
   return ({ dispatch, getState, sourceMaps, client }: ThunkArgs) => {
     const action = { type: "ADD_BREAKPOINT", breakpoint };
     const promise = addBreakpointPromise(getState, client, sourceMaps, action);
     return dispatch({ ...action, [PROMISE]: promise });
   };
 }
 
+export function addHiddenBreakpoint(location: Location) {
+  return ({ dispatch }: ThunkArgs) => {
+    return dispatch(addBreakpoint(location, "", true));
```

Comment:
at this point it would make sense to do `addBreakpoint({location, condition, hidden})`